app/views/notification_mailer/new_course_email.html.erb

```diff
@@ -1,5 +1,10 @@
-<%= t('mailer.new_course', title: @course.title) %>
-<br>
-<%= t('mailer.subscribe_course_html',
-      profile: link_to(t('notifications.profile'),
-                       edit_profile_url)) %>
\ No newline at end of file
+<%= t('mailer.new_course', title: @lecture.title_with_teacher) %>
+<br>
+<%= t('mailer.subscribe_course_html',
+      profile: t('notifications.profile')) %>
```

Comment:
Hm, now it won't work because `mailer.subscribe_course` does not exist in the locale files. Maybe my explanation was not very good. (See [here](https://guides.rubyonrails.org/i18n.html) in 4.4 for an explanation what the `_html` means.)
I would suggest
```
<%= t('mailer.subscribe_course') %>
```
and setting `mailer.subscribe_course` appropriately in the .yml files (without profile paramter), deleting `mailer.subscribe_course_html`